katharsis-core/src/main/java/io/katharsis/jackson/serializer/RelationshipContainerSerializer.java

```diff
@@ -88,53 +87,50 @@ private void writeLinkage(RelationshipContainer relationshipContainer, JsonGener
         Class baseClass = relationshipContainer.getRelationshipField().getType();
         Class relationshipClass = Generics
                 .getResourceClass(relationshipContainer.getRelationshipField().getGenericType(), baseClass);
-        RegistryEntry relationshipEntry = resourceRegistry.getEntry(relationshipClass);
 
         gen.writeFieldName(DATA_FIELD_NAME);
-        writeLinkageField(relationshipContainer, gen, baseClass, relationshipEntry);
+        writeLinkageField(relationshipContainer, gen, baseClass, relationshipClass);
     }
 
-    private void writeLinkageField(RelationshipContainer relationshipContainer, JsonGenerator gen, Class baseClass,
-                                   RegistryEntry relationshipEntry)
+    private void writeLinkageField(RelationshipContainer relationshipContainer, JsonGenerator gen, Class baseClass, Class relationshipClass)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "relationshipClass". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1172)